Python/Product/EnvironmentsList/PipExtensionProvider.cs

```diff
@@ -143,6 +143,11 @@ public sealed class PipExtensionProvider : IEnvironmentViewExtension {
         internal async Task<IList<PipPackageView>> GetInstalledPackagesAsync() {
             string[] args;
 
+            if (!CanExecute) {
+                // Invalid configuration, so assume no packages

```

Comment:
Can the user do something to fix the invalid configuration?  If so we should log to let them know what is wrong.  Assuming of course we know what is wrong.